client/src/components/DataFiles/DataFilesModals/DataFilesAddProject.js

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Formik, Form } from 'formik';
+import FormField from '_common/Form/FormField';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import DataFilesProjectMembers from '../DataFilesProjectMembers/DataFilesProjectMembers';
+
+const DataFilesAddProject = () => {
+  const dispatch = useDispatch();
+  const isOpen = useSelector(state => state.files.modals.addproject);
+
+  const { user } = useSelector(state => state.authenticatedUser);
+
+  // I would like to set the person creating the project as the "Owner" (PI)
+  // Unfortunately, useState does not create a new default state just because 
+  // useSelector came up with a new value for { user }
+  const [ members, setMembers ] = useState([]);
+  // This doesn't work:
+  // const [ members, setMembers ] = useState([ { user, access: "owner" }])
```

Comment:
@jarosenb this commented line doesn't work, and other solutions caused excessive re-render problems. So I had to do a nasty combination of setting two props in the <DataFilesProjectMembers> component